docs/topics/spiders.rst

```diff
@@ -938,6 +979,7 @@ Combine SitemapSpider with other sources of urls:
         def parse_other(self, response):
             pass  # ... scrape other here ...
 
+.. _scrapy-spider-metadata: https://pypi.org/project/scrapy-spider-metadata/
```

Comment:
Maybe we should link to https://scrapy-spider-metadata.readthedocs.io/en/latest/params.html instead.